test/test.wpcom.me.js

```diff
@@ -62,16 +62,21 @@ describe( 'wpcom.me', function() {
 	} );
 
 	describe( 'wpcom.me.groups', function() {
-		it( 'should require groups', done => {
-			me.groups()
-				.then( data => {
-					assert.equal( 'object', typeof data.groups );
-					assert.ok( data.groups instanceof Array );
-
-					done();
-				} )
-				.catch( done );
-		} );
+		// me.groups is a restricted endpoint for Automattic Employees
+		if ( util.testRestrictedEndpoints() ) {
```

Comment:
I feel we could be more graceful about this. Two ideas that come to mind:

- Make this opt-in, not opt-out
- Maybe we can add a new field to the `/oauth2/token-info` response to indicate whether the token is "internal" capable, call it when the test suite starts, and run assertions based on what we'd expect the response to be based on the known token capabilities.